akamai/data/stage/base_rules.json

```diff
@@ -360,7 +360,7 @@
                                         "options": {
                                             "matchOperator": "MATCHES_ONE_OF",
                                             "values": [
-                                                "/beta/rhel"
+                                                "/beta/openshift"
```

Comment:
This redirect will only work for the exact path `/beta/openshift`. Is that intentional?

Reply:
I guess so, in the [doc](https://docs.google.com/document/d/1NkAwv1JT22-iG-erPLMToJ3WGxadjVV8SVg-2j8WxCI) it says: 
Current: /openshift
New: /openshift/clusters

Reply:
Based on @ryelo 's suggestion, this redirect is not needed.